  File "C:\Python27\lib\site-packages\web\application.py", line 239, in process
    return self.handle()
  File "C:\Python27\lib\site-packages\web\application.py", line 229, in handle
    fn, args = self._match(self.mapping, web.ctx.path)
  File "C:\Python27\lib\site-packages\web\application.py", line 477, in _match
    what, result = utils.re_subm('^' + pat + '$', what, value)
  File "C:\Python27\lib\site-packages\web\utils.py", line 531, in re_subm
    compiled_pat = re_compile(pat)
  File "C:\Python27\lib\site-packages\web\utils.py", line 499, in __call__
    update(block=True)
  File "C:\Python27\lib\site-packages\web\utils.py", line 496, in update
    self.running[key].release()
error: release unlocked lock